Devops/fix gh pages mdbook (#227)
sl-sh-dev/sl-sh: README.md

Some of the more prominent features:

* The terminal contains both a shell and a lisp reader so familiar bash-isms like
    ```
    cat file | tr -s " " | cut -d " " -f 2,4
    ```

>   "just work", but starting a command with a lisp form (i.e. a balanced parenthesized s-expression) allows:

Author: ```suggestion
  "just work", but starting a command with a lisp form (an s-expression) allows:
```
